app/main.py

```
@@ -1 +1,20 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Invalid command format."
+                         "Use: cp <source_file> <destination_file>")
```

The `ValueError` message is not properly formatted. The closing parenthesis for `raise ValueError` is missing. It should be `raise ValueError("Invalid command format. Use: cp <source_file> <destination_file>")`.